Jenkinsfile.electron

```diff
@@ -0,0 +1,169 @@
+        }
+      }
+
+    }
+
+    stage('build') {
+
+      steps {
+        script {
+
+          // unlock keychain to ensure build gets signed.
+          withCredentials([string(credentialsId: 'ide-keychain-passphrase', variable: 'KEYCHAIN_PASSPHRASE')]){
+            sh 'security unlock-keychain -p $KEYCHAIN_PASSPHRASE && security set-keychain-settings' // turn off timeout
+          }
+
+          // set requisite environment variables
+          def env = "RSTUDIO_VERSION_MAJOR=${params.RSTUDIO_VERSION_MAJOR} RSTUDIO_VERSION_MINOR=${params.RSTUDIO_VERSION_MINOR} RSTUDIO_VERSION_PATCH=${params.RSTUDIO_VERSION_PATCH} RSTUDIO_VERSION_SUFFIX=${params.RSTUDIO_VERSION_SUFFIX}"
+
+          // build rstudio
+          sh 'cd package/osx && ${env} ./make-package clean --rstudio-target=Electron && cd ../..'
+
+        }
+      }
+
+    }
+
+    stage('tests') {
+
+      environment {
+        PATH = "${env.HOME}/opt/bin:${env.PATH}"
+      }
+
+      steps {
+        script {
+          try {
+            // attempt to run cpp unit tests
+            // problems with rsession finding openssl, so those tests
+            // are disabled until we solve it (#6890)
+            sh "cd package/osx/build/src/cpp && ./rstudio-tests"
+          } catch(err) {
+             currentBuild.result = "UNSTABLE"
+          }
+        }
+      }
+
+    }
+
+    stage('notarize and upload') {
+
+      environment {
+        PATH = "${env.HOME}/opt/bin:${env.PATH}"
+      }
+      
+      steps {
+        script {
+
+          // extract name of package to publish
+          def packageFile = sh (
+            script: "basename `ls src/node/desktop/out/make/RStudio-*.dmg`",
+            returnStdout: true
+          ).trim()
+
+          def buildType = sh (
+            script: "cat version/BUILDTYPE",
+            returnStdout: true
+          ).trim().toLowerCase()
+
+          def buildDest =  "s3://rstudio-ide-build/desktop/macos/"
+
+          withCredentials([usernamePassword(credentialsId: 'ide-apple-notarizer', usernameVariable: 'APPLE_ID', passwordVariable: 'APPLE_ID_PASSWORD')]) {
+            sh "docker/jenkins/notarize-release.sh src/node/desktop/out/make/${packageFile}"
+          }
+
+          // this job is going to run on a macOS slave, which cannot use an instance-profile
+          withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'jenkins-aws']]) {
+            sh "aws s3 cp src/node/desktop/out/make/${packageFile} ${buildDest}"
+          }
+
+          // upload daily build redirects
+          withCredentials([file(credentialsId: 'www-rstudio-org-pem', variable: 'wwwRstudioOrgPem')]) {
+            sh "docker/jenkins/publish-daily-binary.sh https://s3.amazonaws.com/rstudio-ide-build/desktop/macos/${packageFile} ${wwwRstudioOrgPem}"
+          }
+
+          // upload debug symbols to Sentry
+          withCredentials([string(credentialsId: 'ide-sentry-api-key', variable: 'SENTRY_API_KEY')]){
```

Comment:
A "Sentry release" is bound to only one version, so this will also overwrite the symbols for the non-Electron build. The whole Sentry upload step should be commented out.